docs/ja/index.rst

```diff
@@ -10,85 +10,91 @@
    :caption: TECHNICAL DOCUMENTATION
    :hidden:
 
-   Request-Processing
-   Skill-Attributes
-   Response-Building
-   Alexa-Service-Clients
-   Skill-Builders
+   Processing-Request
+   Building-Response
+   Managing-Attributes
+   Calling-Alexa-Service-APIs
+   Configuring-Skill-Instance
    Managing-In-Skill-Purchases
 
-===================
+.. toctree::
+   :caption: REFERENCE
+   :hidden:
+
+   TypeDoc <http://ask-sdk-node-typedoc.s3-website-us-east-1.amazonaws.com/>
+
+*******************
 ASK SDK for Node.js
-===================
+*******************
 
 ASK SDK v2 for
 Node.jsを使うと、ボイラープレートコード（毎回書かなければならないお決まりのコード）を書く手間が不要になります。これにより空いた時間をさまざまな機能の実装に充てることができ、人気のスキルをより簡単に作成できるようになります。
 
 SDKを使って開発を始めるには、以下のリソースをご覧ください。
 
 ガイド
-----------------
+===
 
 `ASK SDKのセットアップ`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-----------------
 
 NPMプロジェクトに依存関係としてSDKをインストールする方法を説明します。
 
 `初めてのスキル開発`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+---------------
 
 Hello Worldサンプルをビルドする手順を詳しく説明します。
 
 `ASK SDK移行ガイド`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-----------------
 
 SDK v1からSDK v2にAlexaスキルを移行する手順を説明します。
 
 技術文書
-----------------
+========
 
 `リクエスト処理`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+------------
 
 リクエストハンドラー、例外ハンドラー、リクエストと応答のインターセプターをビルドする方法を説明します。
 
-`スキルのアトリビュート`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-スキルのアトリビュートを使ったスキルデータの保存と取得の方法を説明します。
-
 `応答のビルド`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-----------
 
 ResponseBuilderを使って、テキスト、カード、オーディオといった複数の要素を使用して1つの応答を構成する方法を説明します。
 
+`スキルのアトリビュート`_
+--------------
+
+スキルのアトリビュートを使ったスキルデータの保存と取得の方法を説明します。
+
 `Alexaサービスクライアント`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+------------------
 
 サービスクライアントを使ってスキルからAlexa
 APIにアクセスする方法を説明します。.
 
 `スキルビルダー`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+----------
 
 スキルインスタンスの構成と作成の方法を説明します。
 
 `スキル内課金（英語）`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+------------------
 
 スキル内課金とスキル内での購入を管理する方法を説明します。
 
 フィードバック
-----------------
+=======
 
 Alexaの機能に関するリクエストや投票は、\ `こちら <https://alexa.uservoice.com/forums/906892-alexa-skills-developer-voice-and-vote>`__\ をご覧ください。
 
 .. _ASK SDKのセットアップ: Setting-Up-The-ASK-SDK.html
 .. _初めてのスキル開発: Developing-Your-First-Skill.html
 .. _ASK SDK移行ガイド: ASK-SDK-Migration-Guide.html
-.. _リクエスト処理: Request-Processing.html
-.. _スキルのアトリビュート: Skill-Attributes.html
-.. _応答のビルド: Response-Building.html
-.. _Alexaサービスクライアント: Alexa-Service-Clients.html
-.. _スキルビルダー: Skill-Builders.html
+.. _リクエスト処理: Processing-Request.html
```

Comment:
This needs updating the file names to new names right? for eg, this one : https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/blob/2.0.x/docs/ja/Request-Processing.rst

Else, this will break the links

Reply:
All the file names have been updated correctly. You can check them through this [link](https://readthedocs.org/projects/ask-sdk-for-nodejstest/).

Reply:
Considering we use the doc blobs from ``ja`` directory, I would suggest to update them to the same ``rst`` names as their ``html`` counterparts, like what was done for ``en`` docs.

Reply:
Those changes are already included in this PR.

Reply:
Got it. Not sure how this was not being shown during the diff changes tab before. 👍